content/rc/how-to/importing-data.md

```diff
@@ -5,61 +5,123 @@ weight: $weight
 alwaysopen: false
 categories: ["RC"]
 aliases: /rv/how-to/importing-dataset-redis-cloud/
+         /rc/how-to/importing-dataset-redis-cloud/
          /rc/how-to/importing-data-database/
          /rv/how-to/importing-data-database/
 ---
-You can import an existing dataset into your Redis Cloud
-instance. This article lists the steps required to share your dataset
-with Redis Cloud.
+You can import an existing dataset into your Redis Cloud instance from an existing Redis server or an RDB file.
 
-You can import a dataset from an existing Redis server or an RDB file
-from various sources detailed below.
+In order to import a sharded database that has multiple RDB files you must first merge the files into a single RDB.
+For assistance, contact [support](https://redislabs.com/support).
 
-You can also import a sharded database that has multiple RDB files by
-merging the files into a single RDB. Contact
-[support](https://redislabs.com/support) if you need help merging the
-RDB files.
-
-Note: Expired keys do not get imported. As a result, the number of keys
-in the source and destination databases might be different after the
-import completes.
+{{% note %}}
+Expired keys are not imported.
+As a result, the number of keys in the source and destination databases can be different after the import is complete.
+{{% /note %}}
 
 ## Import a Dataset from a Redis Server
 
-To import a dataset from any publicly available Redis server, simply
-enter the following information in the Import Dataset dialog of your
-Redis Cloud instance and click the "Import" button when done:
+To import a dataset from any publicly available Redis server:
 
-1. Enter the hostname or the public IP address of the source Redis
-    server in the **Redis Hostname/IP Address** box.
-1. Enter the port of the source Redis server in the Port field, if it
-    is different from the default value of *6379*.
-1. If the source Redis server is set with a password, enter it in the
-    **Password** input control field.
+1. In the Redis Cloud management console, go to the database that you want to import into.
+1. Click **Import**.
+1. In the Source Type field, select **Redis**.
+1. In the **Redis Hostname/IP Address** field, enter the hostname or the public IP address of the source Redis server.
+1. In the **Port** field, enter the port of the source Redis server if it is not the default value of `6379`.
```

Comment:
Consider ending the sentence after "Redis server". Then adding, "By default, the port number is 6379."